cli/src/semgrep/meta.py

```diff
@@ -43,6 +43,11 @@ def git_check_output(command: Sequence[str]) -> str:
                     (fix with `git config --global --add safe.directory $(pwd)`)
 
                 Try running the command yourself to debug the issue.
+
+                Command failed with exit code: {e.returncode}
+                -----
+                Command failed with output:
+                {e.stderr}
```

Comment:
Could we move the stderr near the top of the message? It will be difficult to find the reasons & recommended next step if the command dumps a lot of lines at the end. Please also make sure that `'{command_str}'` and `Try running the command yourself to debug the issue.` stay close to each other.